commit 4443d3b93af3c511e07cfc572803ebfa1f2b1c36
Author: Seth Forshee <seth.forshee@canonical.com>
Date:   Sun May 21 23:36:16 2017 -0500

    UBUNTU: SAUCE: perf callchain: Include errno.h on x86 unconditinally
    
    errno.h is included only if REMOTE_UNWIND_LIBUNWIND is not
    defined, leading to a build error because EINVAL is undeclared.
    
    Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/tools/perf/arch/x86/util/unwind-libunwind.c b/tools/perf/arch/x86/util/unwind-libunwind.c
index 9c917f80c906..ebb5c3e98fea 100644
--- a/tools/perf/arch/x86/util/unwind-libunwind.c
+++ b/tools/perf/arch/x86/util/unwind-libunwind.c
@@ -1,7 +1,7 @@
 // SPDX-License-Identifier: GPL-2.0
+#include <errno.h>
 
 #ifndef REMOTE_UNWIND_LIBUNWIND
-#include <errno.h>
 #include <libunwind.h>
 #include "perf_regs.h"
 #include "../../util/unwind.h"
